### DISCUSSION TITLE
Problem in having a student devloper pack

### DISCUSSION THREAD

**Discussion by lakshit-html** [community/community#143529] [GitHub Education]
### Hi everyone,

i am facing a problem with student devloper pack id pic
![prob](https://github.com/user-attachments/assets/8af19ec3-fdc1-49ef-a6e8-946f39711c66)

**Comment by lakshit-html** [comment]
what is the problem

**Comment by misterob0t** [comment]
Hi @lakshit-html,
I don't know if it can help you but I did it yesterday, I first put my school certificate then I showed it on my webcam (with poor quality) and I think it's is everything.
You must also have the same first and last name in your medical certificate and in your Billing information.
